packages/flutter/lib/src/widgets/app.dart

```diff
@@ -704,6 +750,79 @@ class WidgetsApp extends StatefulWidget {
   /// with "s".
   static bool debugAllowBannerOverride = true;
 
+  // Default shortcuts for the web platform.
+  static final Map<LogicalKeySet, Intent> _defaultWebShortcuts = <LogicalKeySet, Intent>{
```

Comment:
Is the tree-shakaer smart enough to tree-shake these statics out when you're not on the web or macos?